pom.xml

```diff
@@ -1343,8 +1359,33 @@
             <artifactId>sortpom-maven-plugin</artifactId>
           </plugin>
           <plugin>
+            <!-- Sort java imports -->
             <groupId>net.revelc.code</groupId>
             <artifactId>impsort-maven-plugin</artifactId>
+            <version>1.6.2</version>
+            <!--TODO update to latest once we move past JDK11 -->
+            <configuration>
+              <groups>*,java</groups>
```

Comment:
Recommend adding an import group for emissary:
`<groups>emissary,*,java</groups>`